### DISCUSSION TITLE
Can I pull the geysermc plugin from spigot_resources?

### DISCUSSION THREAD

**Discussion by travisboss** [itzg/docker-minecraft-server#1979] [Q&A]
Instead of manually downloading it and adding to my plugins folder, I'd like to keep this up to date and did not want to manually download.

Is there a better solution to this that I am missing?

**Comment by travisboss** [comment]
I get this error when I try.

```
[init] Downloading resource 81297 ...
[init] ERROR: failed to download resource '81297' from https://api.spiget.org/v2/resources/81297/download
[init]        Visit https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/ to pre-download the resource
[init]        instead of using SPIGET_RESOURCES
```

**Reply by itzg** [reply]
It looks like it should work to include `https://ci.opencollab.dev//job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/build/libs/Geyser-Spigot.jar` in the `MODS` env variable. Even though the URL is static, the download will check if it is up to date.
